docs/src/apalache/installation/source.md

```diff
@@ -2,16 +2,17 @@
 
 1. Install `git`.
 2. Install [OpenJDK8][] or [Zulu JDK8][].
```

Comment:
Shall we update the JDK version numbers? I am currently working on Zulu 15, and it's all fine. We should probably check which versions are ok.

Reply:
It looks like JDK 8 is recommended as long as we're on Scala 2.12.0? https://docs.scala-lang.org/overviews/jdk-compatibility/overview.html#version-compatibility-table

Reply:
We are currently using 2.12.15 in `build.sbt`, so this is also outdated :)

Reply:
Oh 😅 I already merged, we will need to follow up on this. Filed #1600

Reply:
We should probably use an LTS release like JDK17 though?

Reply:
true